lib/rules/no-native-reassign.js

```diff
@@ -55,7 +55,7 @@ module.exports = {
             ) {
                 context.report({
                     node: identifier,
-                    message: "{{name}} is a read-only native object.",
+                    message: "Unexpected modification of read-only global variable '{{name}}'.",
```

Comment:
"_Unexpected_" implies that there's a valid "_expected_" way to change the read-only global - is that intentional?

Would this be a clearer message...?
```js
"Read-only global '{{name}}' should not be modified"
```